rust/agama-cli/src/lib.rs

```diff
@@ -153,14 +154,15 @@ async fn finish(manager: &ManagerClient<'_>, method: FinishMethod) -> anyhow::Re
     Ok(())
 }
 
+/// What does this do??
```

Comment:
this code should show progress in case you call CLI when backend is busy...so it writes you that it is busy and show progress. https://github.com/agama-project/agama/pull/2103/files#diff-f6b6409f33ceb3754c4e1f79a4b62d89ee044f73cbf6ca4d29987a4021197fa5R175